plugins/modules/katello_content_view_version.py

```diff
@@ -143,58 +127,34 @@
     state: absent
 '''
 
-RETURN = '''# '''
+RETURN = ''' # '''
 
-try:
-    from nailgun.entities import (
-        ContentViewVersion,
-    )
 
-    from ansible.module_utils.ansible_nailgun_cement import (
-        create_server,
-        find_organization,
-        find_lifecycle_environments,
-        find_lifecycle_environment,
-        find_content_view,
-        find_content_view_version,
-        ping_server,
-        set_task_timeout,
-        naildown_entity_state,
-    )
-    has_import_error = False
-except ImportError as e:
-    has_import_error = True
-    import_error_msg = str(e)
-from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.foreman_helper import KatelloEntityAnsibleModule
 
 
-def promote_content_view_version(module, content_view_version, organization, environments, synchronous, **kwargs):
```

Comment:
Should force be force_promote?

Reply:
The api wants force, but since we control that part, we can let ansible behave the way we like it.

Reply:
I think in the context of this function 'force' is clear.